src/main/java/es/deusto/spq/client/gui/views/auth/EditUserView.java

```diff
@@ -0,0 +1,332 @@
+package es.deusto.spq.client.gui.views.auth;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+
+import javax.swing.JButton;
+import javax.swing.JInternalFrame;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JPasswordField;
+import javax.swing.JTextField;
+import javax.swing.SpringLayout;
+import javax.swing.WindowConstants;
+import javax.swing.text.JTextComponent;
+
+import org.apache.log4j.Logger;
+import org.jetbrains.annotations.Nullable;
+
+import es.deusto.spq.client.controller.HotelManagementController;
+import es.deusto.spq.client.gui.base.View;
+import es.deusto.spq.client.gui.base.ViewManager;
+import es.deusto.spq.client.gui.base.ViewPermission;
+import es.deusto.spq.client.gui.base.ViewType;
+import es.deusto.spq.client.gui.util.SpringUtilities;
+import es.deusto.spq.client.logger.ClientLogger;
+import es.deusto.spq.server.data.dto.UserDTO;
+
+/**
+ * The GUI for editing the user data
+ *
+ * @author egoes
+ *
+ */
+public class EditUserView extends View {
+
+	/**
+	 *
+	 * @param viewManager The class that manages the view
+	 */
+	public EditUserView(final ViewManager viewManager) {
+		super(viewManager);
+		hotelManagementController = viewManager.getClient().getController();
+	}
+
+	private JInternalFrame frame;
+	private final HotelManagementController hotelManagementController;
+
+	// All the form fields and buttons
+	private JTextField nameTextField, emailTextField, phoneTextField, addressTextField;
+	private JPasswordField passwordField, passwordConfirmationField;
+	private JButton saveChanges, dontSave;
+	 private JLabel nameLabel, emailLabel, passwordLabel, passwordConfirmationLabel, phoneLabel, addressLabel;
+
+	// Logger
+	private Logger log;
+
+	@Override
+	public ViewType getViewType() {
+		return ViewType.EDIT_USER;
+	}
+
+	@Override
+	public ViewPermission getViewPermission() {
+		return ViewPermission.NOT_LOGGED_IN;
```

Comment:
Should use `LOGGED_IN`, since we're editing our own user

Reply:
After testing, it seems that by changing this the window doesn't show.

Please set it to NONE and add a TODO so that when I merge #28 I can implement the proper permission level